csrc/quantization/cutlass_w8a8/scaled_mm_entry.cu

```diff
@@ -242,6 +250,28 @@ void get_cutlass_moe_mm_data(
       version_num, ". Required capability: 90");
 }
 
+void get_cutlass_pplx_moe_mm_data(
+    torch::Tensor& expert_offsets, torch::Tensor& problem_sizes1,
+    torch::Tensor& problem_sizes2, const torch::Tensor& expert_num_tokens,
+    const torch::Tensor& non_zero_expert_idxs, const int64_t num_local_experts,
+    const int64_t padded_m, const int64_t n, const int64_t k) {
+  // This function currently gets compiled only if we have a valid cutlass moe
+  // mm to run it for.
+  int32_t version_num = get_sm_version_num();
+#if (defined ENABLE_CUTLASS_MOE_SM90 && ENABLE_CUTLASS_MOE_SM90) || \
+    (defined ENABLE_SCALED_MM_SM100 && ENABLE_SCALED_MM_SM90)
```

Comment:
Should this be the following?
```suggestion
#if (defined ENABLE_CUTLASS_MOE_SM90 && ENABLE_CUTLASS_MOE_SM90)) \
```

I'm not sure why we're looking at `ENABLE_SCALED_MM_SM90`, but the check for `ENABLE_SCALED_MM_SM100` definitely looks wrong